client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesClient.java

```diff
@@ -1059,4 +1061,28 @@ public void putTemplateAsync(PutIndexTemplateRequest putIndexTemplateRequest, Re
         restHighLevelClient.performRequestAsyncAndParseEntity(putIndexTemplateRequest, RequestConverters::putTemplate, options,
             PutIndexTemplateResponse::fromXContent, listener, emptySet());
     }
+
+    /**
+     * Gets index templates using the Index Templates API
+     * <p>
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html"> Index Templates API
+     * on elastic.co</a>
+     */
+    public GetIndexTemplatesResponse getTemplates(GetIndexTemplatesRequest getIndexTemplatesRequest,
```

Comment:
nit: can we call it getTemplate? That's what our SPEC have, without the final 's'

Reply:
I pushed https://github.com/elastic/elasticsearch/pull/31161/commits/7117a3d90eb97f70c6ee7f64be430ec7d5d005ac